pynicotine/gtkgui/transferlist.py

```diff
@@ -134,14 +137,17 @@ def __init__(self, window, transfer_type):
 
         cols["user"].set_sort_column_id(0)
         cols["path"].set_sort_column_id(1)
-        cols["filename"].set_sort_column_id(2)
-        cols["status"].set_sort_column_id(3)
-        cols["queue_position"].set_sort_column_id(13)
-        cols["percent"].set_sort_column_id(5)
-        cols["size"].set_sort_column_id(10)
-        cols["speed"].set_sort_column_id(12)
-        cols["time_elapsed"].set_sort_column_id(14)
-        cols["time_left"].set_sort_column_id(15)
+        cols["file_type"].set_sort_column_id(2)
```

Comment:
It's debatable if sorting by the label value of FILE_TYPE_ICON_LABELS rather than the actual file extension is desirable or not. The sort order seems to be a bit random.

Perhaps it would be useful if the tooltip did show the extention, such as "mp3 (Audio)".

Reply:
I personally like having the files in generic file type groups when sorting, it's easier to separate music and videos from images and other documents/text files.